### ISSUE TITLE
extra space

### ISSUE THREAD

**Issue by interlab** [Raku/doc#3151]
Url: https://docs.raku.org//type/IO::Socket::Async

Code highlighting seems to add an extra space? See screenshot:

![space](https://user-images.githubusercontent.com/2246621/71929592-9efc0180-31a2-11ea-820a-d92a6c22f687.png)

**Comment by Altai-man** [comment]
Yes, the space is not present in the original file, so it looks like a syntax highlighter bug.
Thanks for reporting! I have submitted an issue to code highlighter package the docs use.

**Comment by coke** [comment]
@Altai-man  - can you provide a link to the ticket in the other system?

**Comment by Altai-man** [comment]
Sure, https://github.com/perl6/atom-language-perl6/issues/92
